_schema.graphql

```diff
@@ -9,7 +9,7 @@ input AddAssetToConsignmentSubmissionInput {
   """
   clientMutationId: String
   geminiToken: String!
-  submissionID: ID!
+  submissionId: ID!
```

Comment:
Is this gonna blow anything up?

Reply:
(`D` to `d`

Reply:
Yeah, I’m unclear about what’s safe/risky here so could use some direction. What I should probably do is "smoke test" this locally with these changes pushed into a local copy of MP. 🤔

Reply:
@damassi so if I'm understanding you correctly, I think what we want is for this file to have no diff, correct? I guess I was hoping for an opportunity to switch from snake to camel case but maybe that would break the app?

Reply:
Not necessarily! What we need to be concerned about is what _has been used_ in the app -- everything else can be changed. I was looking at these files in particular: 

https://github.com/artsy/eigen/tree/master/src/lib/Components/Consignments/Submission

Which contain all of the queries. 

The thinking is backward compat, but changes made to unused areas of the codebase are fair game. (We're lucky, because the surface area of this API use is small, so we can be fairly confident of what has been used.)

Reply:
Ok makes total sense now, thanks!